internal/resources/resource_volume.go

```diff
@@ -326,6 +371,26 @@ func resourceMetalVolumeRead(d *schema.ResourceData, meta interface{}) (err erro
 		return fmt.Errorf("set Size: %v", err)
 	}
 
+	if err = d.Set(vSizeInUse, math.Round(float64(volume.CapacityUsed)/KiBToGBConversion)); err != nil {
```

Comment:
Why is the unit defined differently here vs client?

Reply:
I did this to align with the existing `Size` entry in the volume schema. 
The `Size` is getting converted from KiB to GB here so had to do the same for the `SizeInUse`

GB is more understandable/usable I think.
We will have to change both fields to align with the client. Let me know your thoughts on that.

Reply:
Ok thanks. Keeping vSize and vSizeInUse units the same is good.